bundle-workflow/src/test_workflow/perf_test_cluster.py

```diff
@@ -0,0 +1,78 @@
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+
+from test_workflow.test_cluster import TestCluster
+
+
+class Cluster:
```

Comment:
This should be a capability of any cluster, so `PerfTestCluster` would inherit `TestCluster` and the latter has a static `def create(cls, ...)` method that is implemented by calling `cls._init_`...

You can refactor this later.